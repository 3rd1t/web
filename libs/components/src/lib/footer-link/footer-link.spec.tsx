import React from "react"
import { render } from "@testing-library/react"
import renderer from "react-test-renderer"
import FooterLink from "./footer-link"

describe(" FooterLink", () => {
  it("should render successfully", () => {
    const { baseElement } = render(
      <FooterLink hoverColor="text-white" href="#">
        Something
      </FooterLink>,
    )
    expect(baseElement).toBeTruthy()
  })

  it("renders correctly", () => {
    const tree = renderer
      .create(
        <FooterLink hoverColor="text-white" href="#">
          Something
        </FooterLink>,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
