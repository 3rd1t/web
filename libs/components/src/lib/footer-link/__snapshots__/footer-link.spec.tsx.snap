// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` FooterLink renders correctly 1`] = `
<a
  className="block focus:outline-none hover:text-white"
  href="#"
>
  Something
</a>
`;
